build/builtins.js

```diff
@@ -0,0 +1,20 @@
+/*
+** Core logic from https://github.com/sindresorhus/builtin-modules
+** Many thanks to @sindresorhus
+*/
+const { builtinModules } = require('module')
+
+const blacklist = [
+  'sys'
+]
+
+const builtins = (builtinModules || Object.keys(process.binding('natives')))
+  .filter(x => !/^_|^(internal|v8|node-inspect)\/|\//.test(x) && !blacklist.includes(x))
+  .sort()
+
+export const builtinsMap = () => builtins.reduce((obj, builtin) => {
```

Comment:
Could we leverage `Map` here? :thinking:

Reply:
Actually I wrote it with Map at the beginning, but object is handy and also map structure underlying,so I changed.